src/views/preview/project-view.jsx

```diff
@@ -379,9 +379,10 @@ class Preview extends React.Component {
                     }
 
                     if (showAlerts) {
-                        // Check for username block only if user is logged in
+                        // Check for username and video blocks only if user is logged in
                         if (this.props.isLoggedIn) {
                             newState.showUsernameBlockAlert = helpers.usernameBlock(projectData[0]);
+                            newState.showCloudDataAndVideoAlert = hasCloudData && helpers.videoSensing(projectData[0]);
```

Comment:
What makes this message go away -- won't it just persist as long as the project includes both cloud vars and video sensing?

Reply:
Thanks, OK, that makes sense! I think I'll need to try it to see how it feels, but that's what the bug hunt is for!